runner/ansible/roles/1.3.4/tasks/main.yml

```diff
@@ -0,0 +1,23 @@
+---
+
+- name: Set Test ID
+  set_fact:
+    test_id: '1.3.4'
+
+- name: "{{ test_id }}.check"
+  shell: |
+    echo -n "SBD_COUNT="
```

Comment:
What is this first `echo` for?
Here the purpose is to test if we have 3 (expected value) SBD devices configured. 
For that, it checks the `/etc/sysconfig/sbd` file, and withing this the `SBD_DEVICE` entry.
This entry can be like this:
`SBD_DEVICE=/dev/vdb;/dev/vdc;/dev/ddd` (the value can go even within quotes, and have spaces after `;` value).
So in this case, we would count 3 (split the value with `;`)

Reply:
The audit for this one from the example is:
        audit: |
          sbdarray=$(grep -E '^SBD_DEVICE=' /etc/sysconfig/sbd  | grep -oP 'SBD_DEVICE=\K[^.]+' | sed 's/\"//g')
          IFS=';' sbdarray=( $sbdarray )
          echo "SBD_COUNT=${#sbdarray[@]}"
        tests:
          test_items:
            - flag: "SBD_COUNT"
              compare:
                op: eq
                value: 3

So this translated to test for 
"SBD_COUNT=3"
which is what is in the vars/azure_env.yml 

I will need to use the counting hack in 1.3.7 here to get the count right. "${#" just messes up jinja in  ansible.